compiler/ilgen/VirtualMachineOperandStack.cpp

```diff
@@ -130,6 +131,18 @@ VirtualMachineOperandStack::Commit(TR::IlBuilder *b)
       }
    }
 
+// Update the OperandStack_base and _stackTopRegister after the Virtual Machine moves the stack.
+// This call will normally be followed by a call to Reload if any of the stack values changed in the move
+void
+VirtualMachineOperandStack::UpdateStack(TR::IlBuilder *b, TR::IlValue *stack)
+   {
+   b->Store("OperandStack_base", stack);
+   // Adjust the vm _stackTopRegister by number of elements that have been pushed onto the stack.
+   // _stackTop is -1 at 0 pushes, 0 for 1 push, so # of elements to adjust by is _stackTop+1
+   _stackTopRegister->Store(b, stack);
+   _stackTopRegister->Adjust(b, (_stackTop+1)*_pushAmount);
```

Comment:
You are correct.  I thought that the _stackTopRegister would need to be updated to Reload to work but it does not.  Removed those two lines.